exercise-core/src/main/java/com/examples/attsw/exercise/core/model/Employee.java

```diff
@@ -2,14 +2,25 @@
 
 public class Employee {
 
+	private String name;
 	private String id;
 
-	public Employee(String id) {
+	public Employee(String name, String id) {
+		this.name = name;
 		this.id = id;
 	}
-	
+
+	public String getName() {
+		return name;
+	}
+
 	public String getId() {
-		return this.id;
+		return id;
+	}
+
+	@Override
+	public String toString() {
+		return getName();
```

Comment:
also the id should be part of the String. Please have a look at the Java conventions for the toString method, or, simply have Eclipse generate the toString method